e2echart/e2e-chart-template.html

```diff
@@ -97,13 +97,50 @@ <h5 class="modal-title">Resource</h5>
         return false
     }
 
+    // If this is a leader found, lost, elected, or missing, return true and the reason.
+    // Otherwise, return false.
+    function isEtcdLeaderFoundOrLost(eventInterval) {
+        const { tempStructuredLocator, tempStructuredMessage } = eventInterval;
```

Comment:
Whoah, how does this work?

Reply:
it's a destructuring assignment (kind of a shorthand in javascript) -- but no longer necessary.